templates/article.html

```diff
@@ -73,6 +73,7 @@ <h1>{{ title }}</h1>
             </div>
             {% endif %}
 
+            <applause-button style="width: 50px; height: 50px;"/>
```

Comment:
You should make the applause button optional.

We let user disable a feature on site wide and per article basis.

See the how we offer two ways to enable and disable comments.

https://elegant.oncrashreboot.com/enable-utterances-comments

Reply:
Done. I've tried to put `applause_button: False` in one of the article, and it works.